src/commands/install.yml

```diff
@@ -12,6 +12,10 @@ parameters:
       but before verifying Cypress and saving cache.
     type: string
     default: ""
+  install-browsers:
```

Comment:
nit-pick here but can you set the description as the first item in all the params for consistency?